helper_scripts/analyze.py

```diff
@@ -50,8 +50,12 @@ def analyze_kafka_dump(directory):
                              order_cost[merchant_id], profit[merchant_id]])
 
     create_inventory_graph(directory, merchant_id_mapping)
-    create_profit_per_minute_graph(directory, merchant_id_mapping)
-    create_revenue_per_minute_graph(directory, merchant_id_mapping)
+    create_line_chart(directory, merchant_id_mapping,
+        topic='profitPerMinute', value_name='profit', label='Profit per Minute')
+    create_line_chart(directory, merchant_id_mapping,
+        topic='revenuePerMinute', value_name='revenue', label='Revenue per Minute')
+    create_line_chart(directory, merchant_id_mapping,
+        topic='profit', value_name='profit', label='Profit')
```

Comment:
Can you name `cumulativeProfit` or something similar? With `profit` I would be expecting a simple bar chart of the final profit and not profit over time.